android/assets/jsons/Civ V - Gods & Kings/Policies.json

```diff
@@ -84,7 +84,8 @@
                 "name": "Republic",
                 "uniques": [
                     "[+1 Production] [in all cities]",
-                    "[+5]% Production when constructing [All] buildings [in all cities]"
+                    "[+5]% Production when constructing [All] buildings [in all cities]",
+                    "[+5]% Production when constructing [All] wonders [in all cities]"
```

Comment:
Needs confirmation from Civ V. There's no real documentation that this is supposed to actually give production for wonders

Reply:
And there's basically no guides or whatever that suggests this does give production to wonders, unlike other niche things